### PR TITLE
English to Swedish

### LINKED ISSUE #718: Change English on website to Swedish
## ✨ Feature request
Currently different titles and descriptions are mixed with the both languages. Since the website is developed for use by Swedes the website should be in Swedish. 

### 📜 Acceptance Criteria
Everything readable by the user should be in Swedish, except routes.

### PR DESCRIPTION
**Pull request by ntijoh-axel-ostan**
## ⚙️ What issue did you implement or fix?
Fixade all engelsk text till svenska 

## 📜 Summary
- Fixade all text.

## 📝 Type of change
text changes

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 This change requires a documentation update

## ✅ Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
- Change bar into zar. :shipit:

close #718, #713

### CODE REVIEW

File: frontend/src/components/LoanFormComponent.vue [hunk 1]
```diff
@@ -69,7 +69,7 @@
           <template v-slot:activator="{ on, attrs }">
             <v-text-field
               v-model="form.expiration_date"
-              label="Expire date"
+              label="Datum för återlämning"
```
Comment (ntijoh-evelina-karlsson) [inline frontend/src/components/LoanFormComponent.vue:72]: Bytte 'Återlämnings datum' till 'Datum för återlämning'

File: frontend/src/components/ReturnLoanComponent.vue [hunk 2]
```diff
@@ -52,7 +52,7 @@
               large
               @click="onSubmit"
             >
-              Return
+              Lämna tillbaka
```
Comment (ntijoh-evelina-karlsson) [inline frontend/src/components/ReturnLoanComponent.vue:55]: Bytte 'Läman tillbaka' till 'Lämna tillbaka'

File: frontend/src/views/LibraryView.vue [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <template>
   <v-container> 
     <ReturnLoanComponent />
-    <h2>Add loan</h2>
+    <h2>Lägg till bock att låna</h2>
```
Comment (ntijoh-evelina-karlsson) [inline frontend/src/views/LibraryView.vue]: Ändrade 'bock' till 'bok'

**Review by ntijoh-evelina-karlsson — APPROVED** [review]
Hittade flertalet stavfel men det ska vara löst nu!

**Review by ntijoh-linus-sjunnesson — APPROVED** [review]
gœtt

**Pull request merged**